docs/source/examples/batch-inference.rst

```diff
@@ -0,0 +1,315 @@
+.. _offline-batch-inference:
+
+Large-Scale Offline Batch Inference
+===================================
```

Comment:
Overall may need a figure to illustrate all concepts and how they relate: chunks / work items (is one item = one jsonl in this case?) / groups / workers, ..

At least a bullet list at the top to explain these. I'm currently confused by the above.

Reply:
Reduced the concepts to chunks and workers. Do you think that is better?